src/writeData/components/WriteDataHelperTokens.tsx

```diff
@@ -102,7 +105,11 @@ const WriteDataHelperTokens: FC = () => {
         className="write-data--details-widget-title"
       >
         Token
-        <GenerateTokenDropdown />
+        {isFlagEnabled('tokensUIRedesign') ? (
```

Comment:
I think we can remove this `isFlagEnabled` because we have this flag on in Production and I am right now in the process of removing this flag from everywhere. So, we can just render `GenerateTokenDropdownRedesigned`. What do you think?

Reply:
I can do that! Do you think from a commit-history point, would it be better to remove this part after we've officially removed the flag? Just in case?

Reply:
Hmm yeah, let's commit this, and I can pull latest and remove the flag in my pr :)

Reply:
Or, maybe I can just wait until your flag removal PR is in and then merge this code without the flag? Let me know what you think!